src/utils/workflow_utils/inc/aduc/workflow_utils.h

```diff
@@ -233,20 +233,20 @@ size_t workflow_get_update_files_count(ADUC_WorkflowHandle handle);
  *
  * @param handle A workflow data object handle.
  * @param index An index of the file to get.
- * @param entity An output file entity object. Caller must free the object with workflow_free_file_entity().
- * @return true If succeeded.
+ * @param entity An output file entity object. Caller must uninitialize it via ADUC_FileEntity_Uninit when done.
```

Comment:
This PR is a great start, but if you notice, the bug title I created is "workflow_get*" -- there's a lot of other methods in here that are returning ADUC_FileEntity**.

any chance of getting those changed as well?

Reply:
I had missed that. In that case, this is a larger refactoring task that I can continue on post .2 patch.

Reply:
can you create a new bug/task for that?

Reply:
I went ahead and fixed remaining--wasn't too bad.